tests/jballerina-unit-test/src/test/resources/test-src/closures/closures-in-resource.bal

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/lang.test;
+
+public type TLift record {|
+    readonly string id;
+    string name;
+    string status;
+    int capacity;
+    boolean night;
+    int elevationgain;
+    TTrail trailAccess;
+|};
+
+public type TTrail record {|
+    string id;
+    string name;
+    string status;
+    string difficulty;
+    boolean groomed;
+    boolean trees;
+    boolean night;
+|};
+
+listener test:MockListener echoEP = new(9090);
+
+service /testService on echoEP {
+
+    resource function get closureTest1(string status) returns string {
+        var addFunc = function (int funcInt) returns (string) {
+            return status;
+        };
+        return addFunc(1);
```

Comment:
We should probably try and add a test where this resource is invoked, right? @rdhananjaya, do we have such tests?

Reply:
https://github.com/ballerina-platform/ballerina-lang/blob/fe42ba2cf2ed315f6aca8c8f1431276282384bba/tests/jballerina-unit-test/src/test/resources/test-src/bala/test_bala/object/test_service_objects.bal#L88

Reply:
Also this
https://github.com/ballerina-platform/ballerina-lang/blob/fe42ba2cf2ed315f6aca8c8f1431276282384bba/tests/jballerina-unit-test/src/test/resources/test-src/services/service_decl.bal#L208

Reply:
@MaryamZi, @rdhananjaya changed the test to have resource invocation (42c1683).